File: solutions/C/darts/1/darts.c
#include "darts.h"
#include <math.h>

uint8_t score(coordinate_t coordinate) {
    float radius = sqrt(coordinate.x * coordinate.x + coordinate.y * coordinate.y);
    if (radius <= 1.0f) {
        return 10;
    }
    if (radius <= 5.0f) {
        return 5;
    }
    if (radius <= 10.0f) {
        return 1;
    }
    return 0;
}

File: solutions/Python/darts/3/darts.py
def score(x, y):
    distance = ((x * x ) + (y * y)) ** (1 / 2)
    if distance <= 1:
        return 10
    if distance <= 5 :
        return 5
    if distance <= 10:
         return 1 
    return 0
   
    
     
